src/components/relationships-panel/package-lock.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@bcrs-shared-components/relationships-panel",
+  "version": "0.0.0",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "@bcrs-shared-components/relationships-panel",
+      "version": "0.0.0"
+    }
+  }
+}
```

Comment:
The other components in this repo don't have a package lock file, so yours shouldn't either.

Reply:
Removed the file.